extensions/http/src/main/java/org/mule/extension/http/internal/listener/HttpListener.java

```diff
@@ -229,8 +229,10 @@ public void onStart(SourceCallback<Object, HttpRequestAttributes> sourceCallback
     startIfNeeded(responseFactory);
 
     validatePath();
-    this.parseRequest = config.resolveParseRequest(configurationOverrides.getParseRequest());
-    if (config.resolveInterpretRequestErrors(configurationOverrides.getInterpretRequestErrors())) {
+    // this.parseRequest = config.resolveParseRequest(configurationOverrides.getParseRequest());
```

Comment:
Why the comments? If the config overrides makes that obsolete can we delete that and the methods used there?